Lesson05/HW05.SchoolControls/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+namespace HW05.SchoolControls
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            
+            while (true)
+            {
+                Console.WriteLine("Введите номер контрольного теста");
+                int numtest = 0;
+                numtest = Convert.ToInt32(Console.ReadLine());
+
+
+                if ((numtest >= 1) && (numtest <= 15))
```

Comment:
Give readable variable names and read the rules for naming variables again.

Reply:
done